src/server/worker/pipeline/transform/transform_test.go

```diff
@@ -191,21 +173,28 @@ func mockBasicJob(t *testing.T, env *testEnv, pi *pps.PipelineInfo) (context.Con
 			cancel()
 		}
 	}
-
 	env.MockPPSTransactionServer.UpdateJobStateInTransaction.Use(func(txnCtx *txncontext.TransactionContext, request *pps.UpdateJobStateRequest) error {
 		updateJobState(request)
 		return nil
 	})
-
 	env.MockPachd.PPS.UpdateJobState.Use(func(ctx context.Context, request *pps.UpdateJobStateRequest) (*types.Empty, error) {
 		updateJobState(request)
 		return &types.Empty{}, nil
 	})
 
+	eg, ctx := errgroup.WithContext(ctx)
+	reg, err := newRegistry(env.driver, env.logger)
+	require.NoError(t, err)
+	eg.Go(func() error {
+		if err = reg.startJob(proto.Clone(jobInfo).(*pps.JobInfo)); err != nil {
+			return err
+		}
+		return nil
```

Comment:
is this just because the linter was complaining? iirc you can just do `_ = reg.startJob(...)` and it will be fine